types/storybook__react/index.d.ts

```diff
@@ -12,15 +13,21 @@ import * as React from 'react';
 export type Renderable = React.ComponentType | JSX.Element;
 export type RenderFunction = () => Renderable | Renderable[];
 
+export interface DecoratorParameters {
+    [key: string]: any;
+}
```

Comment:
Here's where we see parameters being used as an object with the `[key: string]: any` signature: https://github.com/storybooks/storybook/blob/next/lib/core/src/client/preview/client_api.js#L112-L120